app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> str:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:

```

Comment:
Code Style: It's a good practice to use type annotations. Please annotate the `visitor` argument as well.